cogs/pokemon.py

```diff
@@ -539,8 +539,8 @@ async def order(self, ctx, *, sort: str = ""):
 
         sort = sort.lower()
 
-        if sort not in ("number", "iv", "level", "pokedex"):
-            return await ctx.send("Please specify either `number`, `IV`, `level`, or `pokedex`.")
+        if sort not in [a + b for a in ("number", "iv", "level", "pokedex") for b in ("+", "-", "")]:
```

Comment:
I initially planned to make the list into a separate variable `options` and then use that in both the if statement and the error message with .join(). Ended up sticking with the original "format" though, which do you think would've been better?